### PR TITLE
feat(PPDSC-2135): add unified script to Consent

### PR DESCRIPTION
**Pull request by tbradbury**
PPDSC-2135

**What**

1. Sourcepoint have a new unified script, all client created after May 21 need to use this new script.
2. Added a new unified script option in the consent component. Also updated the docs.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2135-sourcepoint 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2135-sourcepoint/storybook

File: site/pages/_document.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default class MyDocument extends Document<Props> {
           {helmet.script.toComponent()}
           <HTMLMeta />
           <Consent
-            sourcePointConfigNonTCFV1={{
+            sourcePointConfigUnifiedTCF={{
```
Comment (Xin00163) [inline site/pages/_document.tsx:60]: I don't think it's working on the doc site. If you click on `[Privacy policy](http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2135-sourcepoint/#)`, nothing is shown

Comment (tbradbury) [reply site/pages/_document.tsx]: 👍 I've update the consent cookie link and it now works.

**Review by Xin00163 — COMMENTED** [review]
Just to be on the safe side: does this need to be verified by clients like virgin for example?

**Review by mutebg — APPROVED** [review]

File: cypress/support/commands.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Cypress.Commands.add('mockConsentAndVisit', url => {
-  cy.intercept('GET', '**/wrapper/tcfv2/v1/gdpr/**', 'fx:consent-request');
+  cy.setCookie('nukt_sp_consent', 'JABCDEFGHI');
+  cy.setCookie('consentUUID', '0714192a-0776-4b78-b462-41dde53d35e8_7');
```
[the remark this answers is not in the document]
Comment (tbradbury) [reply cypress/support/commands.js:3]: I don't believe so it what we do on Render and we have not had to change them once.

**Comment by tbradbury** [comment]
> Just to be on the safe side: does this need to be verified by clients like virgin for example?

Yes but Wireless can't test till at least the 30th. This does not effect the old implementation so James says we should move it forward and Wireless can test when ready

File: site/pages/components/consent-settings-link.mdx [hunk 3]
```diff
@@ -29,13 +29,27 @@ The consent settings link supports any prop valid on the <a href='/components/li
...
+    <br/>
+    The legacy TCFv2 script only needs privacy manager id.
+  </Prop>
+  <Prop name="gdpr" type="boolean">
+    If your using the unified sourcepoiont script for GDPR.
+  </Prop>
+  <Prop name="ccpa" type="boolean">
+    If your using the unified sourcepoiont script for CCPA.
+  </Prop>
+  <Prop name="tabToOpen" type="string">
```
[the remark this answers is not in the document]
Comment (tbradbury) [reply site/pages/components/consent-settings-link.mdx]: updated

**Review by Xin00163 — APPROVED** [review]

File: site/pages/components/consent-settings-link.mdx [hunk 4]
```diff
@@ -29,13 +29,27 @@ The consent settings link supports any prop valid on the <a href='/components/li
...
     Your privacy manager id.
   </Prop>
+  <Prop name="siteId" type="string">
+    Your Sourcepoint site id. This is used for the legacy TCFv1 sourcepoint script.
+    <br/>
+    <br/>
+    The legacy TCFv2 script only needs privacy manager id.
+  </Prop>
+  <Prop name="gdpr" type="boolean">
+    If your using the unified sourcepoiont script for GDPR.
```
Comment (mstuartf) [inline site/pages/components/consent-settings-link.mdx:42]: Very minor but a few typos: `you're` and `Sourcepoint`.

File: site/pages/components/consent-settings-link.mdx [hunk 5]
```diff
@@ -29,13 +29,27 @@ The consent settings link supports any prop valid on the <a href='/components/li
...
+  </Prop>
+  <Prop name="ccpa" type="boolean">
+    If your using the unified sourcepoiont script for CCPA.
```
Comment (Vanals) [inline site/pages/components/consent-settings-link.mdx:45]: typo here as well, @mstuartf left you a comment too above

**Review by Xin00163 — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Pull request merged**